db/schema.rb

```diff
@@ -29,6 +29,8 @@
     t.datetime "created_at",   null: false
     t.datetime "updated_at",   null: false
     t.integer  "affiliate_id"
+    t.string   "phone_number"
+    t.date     "dob"
```

Comment:
are we sure if we need the coordinator's personal info(i.e:dob?)
if we do, we can call this field `date_of_birth`, it's more informative.